src/controllers/__tests__/products.test.js

```diff
@@ -104,21 +105,38 @@ describe('Products routes', () => {
         type: 'Product',
         location: 'Ankara',
         condition: 'Used',
+        seller: createdUser.body.data._id,
       };
       myproduct1 = await Product.create(product1);
       myproduct2 = await Product.create(product2);
     });
 
     test('Update given field, return with 200 status code', async () => {
-      const toUpdate = { title: 'Potato' };
+      const toUpdate = {
+        title: 'potato',
+        description: 'Matehs',
+        price: 120,
+        category: 'Book',
+        coverImage: 'www.book.jpg',
+        type: 'Product',
+        location: 'Istanbul',
+        condition: 'Used',
+      };
+
       const res = await server
-        // eslint-disable-next-line no-undef
         .patch(`/api/products/${myproduct1._id}`)
         .send(toUpdate);
       expect(res.status).toBe(200);
       expect(res.headers['content-type']).toMatch('application/json');
       expect(res.body.success).toBe(true);
-      expect(res.body.data.title).toBe('Potato');
+      expect(res.body.data.title).toBe('potato');
+      expect(res.body.data.description).toBe('Matehs');
+      expect(res.body.data.price).toBe(120);
+      expect(res.body.data.category).toBe('Book');
+      expect(res.body.data.coverImage).toBe('www.book.jpg');
+      expect(res.body.data.type).toBe('Product');
+      expect(res.body.data.location).toBe('Istanbul');
+      expect(res.body.data.condition).toBe('Used');
```

Comment:
Instead of checking every field inside `res.body.data`, you can check if `res.body.data` contains `toUpdate` object. 
Such as:

`expect(res.body.data).toEqual(expect.objectContaining(toUpdate));`